src/components/Container/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import Heading from 'components/Heading';
+import Spinner from 'components/Spinner';
+import Error from 'components/Error';
+
+export const AsyncContainer = ({ loading, error, children }) => {
```

Comment:
It's an internal component; I don't think the name is super important. Async makes more sense anyways since it has to do with loading and errors.